pyproject.toml

```diff
@@ -1,3 +1,27 @@
 [build-system]
-requires = ["setuptools>=43", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry_core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "unpaddedbase64"
+version = "2.0.0rc3"
```

Comment:
Why rc3 here?

Reply:
rc2 was when I tried out Flit, which was also a very nice tool. But then I tried Poetry again and... I really, really like Poetry. ;)

Reply:
That's fair, I suppose I'm just wondering why we need to consider rc2 if we never released/merged it.

Reply:
Fair enough. Heck, let's call it all rc1 since this is all on an isolated branch right now anyway

Reply:
Rewrote the tipmost commit to call this rc2